tags/interceptors_test.go

```diff
@@ -116,9 +138,10 @@ func (s *TaggingSuite) TestPingList_WithCustomTags() {
 		}
 		require.NoError(s.T(), err, "reading stream should not fail")
 		tags := tagsFromJson(s.T(), resp.Value)
+		assert.Contains(s.T(), tags, "grpc.request.deadline", "the tags should contain a deadline")
 		assert.Contains(s.T(), tags, "peer.address", "the tags should contain key address")
```

Comment:
Maybe not `key address` but just `an address` like in message above? Key is `peer.address`.